tests/lib/netapplier_test.py

```diff
@@ -178,38 +178,3 @@ def test_edit_existing_bond(netinfo_nm_mock, netapplier_nm_mock):
     m_apply_changes.assert_called_once_with(
         desired_config[INTERFACES], State(desired_config)
     )
-
-
-@mock.patch.object(netapplier, "_apply_ifaces_state", lambda *_: None)
-def test_warning_apply():
-    with pytest.warns(FutureWarning) as record:
```

Comment:
Please change this to expect an exception instead of removing the test, also in the other instances.